logger/sql.go

```diff
@@ -13,6 +13,12 @@ import (
 	"gorm.io/gorm/utils"
 )
 
+const (
```

Comment:
The changes in this file are unrelated to the functional options, maybe it should be in a separate PR?

Reply:
Ah, I see #4059 does that. Maybe this branch should be rebased to remove this commit from it?

Reply:
Yes,here I have mentioned a PR again